packages/docker-engine/docker.service

```diff
@@ -10,7 +10,7 @@ StartLimitIntervalSec=60s
 
 [Service]
 Type=notify
-ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
+ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock --default-ulimit nofile=1024:4096
```

Comment:
Can we put this in `daemon.json` instead?  That's where we have other configuration, modulo the 'containerd' setting that seems like it can't go in the file.  (The -H setting seems like it could have been in the file?)

https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file

It'll make it easier to configure for users, and easier to find all relevant settings.